packages/launcher/test/unit/windows_spec.ts

```diff
@@ -25,6 +25,10 @@ function stubBrowser (path: string, version: string) {
   .resolves(true)
 }
 
+function doubleEscape (s: string) {
+  return win32.join(...s.split(win32.sep)).replace(/\\/g, '\\\\')
+}
+
```

Comment:
could you just import this from `windows.ts`?

Reply:
I think you mean `packages/launcher/lib/windows/index.ts`

Exported that function, used it in the test case and added its own test cases